SPECS/nvidia-container-toolkit/nvidia-container-toolkit.signatures.json

```diff
@@ -1,6 +1,6 @@
 {
   "Signatures": {
     "nvidia-container-toolkit-1.17.3-vendor.tar.gz": "894d10f0504e7a8a8fe8748d736288e90f23e24f13aac058b6b83c4ca99dc40f",
```

Comment:
The vendor source tar needs to be upgraded based on the latest source of nvidia container toolkit v1.17.4. By checking the upstream repo of nvidia container toolkit, there are changes being made to the vendor source between the v1.17.4 release and the last v1.17.3 release.